README.md

```diff
@@ -37,7 +37,7 @@ import Document, {
   Main,
   NextScript,
 } from "next/document";
-import { trackingHeadScript, trackingBodyScript } from "@phntms/react-gtm";
+import { TrackingHeadScript, TrackingBodyScript } from "@phntms/react-gtm";
 
 const GA_TRACKING_ID = process.env.NEXT_PUBLIC_GA_TRACKING_ID || "";
```

Comment:
I would make this non next specific.

Reply:
Agreed, I'll update down the road at some indication.